### PR TITLE
scheduler date opened

### PR DESCRIPTION
**Pull request by proteusvacuum**
Returns the date that the case was opened if the first scheduled visit has already passed (so that new cases don't show up as immediately late, and the first scheduled visit is the date that the case was created).

Also, re-enables form workflows. 

Code buddy: @orangejenny 
ping: @sheelio

### CODE REVIEW

File: corehq/apps/app_manager/suite_xml.py [hunk 1]
```diff
@@ -1400,6 +1400,7 @@ def _schedule_detail_variables(module, detail, detail_column_infos):
         has_schedule_columns = any(ci.column.field_type == FIELD_TYPE_SCHEDULE for ci in detail_column_infos)
         has_schedule = getattr(module, 'has_schedule', False)
         if (has_schedule and module.all_forms_require_a_case() and has_schedule_columns):
+            yield DetailVariable(name=SCHEDULE_DATE_CASE_OPENED, function="date_opened")  # date case is opened
```
Comment (orangejenny) [inline corehq/apps/app_manager/suite_xml.py]: i am generally all about comments, but this one is perhaps not adding much

**Comment by orangejenny** [comment]
looks good once tests pass

File: corehq/apps/app_manager/suite_xml.py (hunk 1 above)
Comment (proteusvacuum) [inline corehq/apps/app_manager/suite_xml.py]: Hah, you are right. I actually had a different, less descriptive variable name, which I then changed. :smile_cat:

**Comment by proteusvacuum** [comment]
:white_check_mark:

**Pull request merged**